File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseExternal.java
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.documents.core.network.common.models.BaseResponse;

public class ResponseExternal extends BaseResponse {

    @SerializedName(BaseResponse.KEY_RESPONSE)
    @Expose
    private String response;

    public String getResponse() {
        return response;
    }

    public void setResponse(String response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseExternal.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseExternal(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: String? = null
) : BaseResponse()